### ISSUE TITLE
Is there a way to stop a script after a certain amount of commands have been performed?

### ISSUE THREAD

**Issue by TMP7ML** [Fate-Grand-Automata/FGA#1862]
### Preparation

- [X] I read the [Troubleshooting Guide](https://github.com/Fate-Grand-Automata/FGA/wiki/Troubleshooting)
- [X] My question isn't answered in any other issues (even the closed ones)

### Your question

Lets say I want to manually play after I activated all my servants skills on turn 1, or after I launched my servant's NP in turn 1 and want to manually play from turn 2 onwards.
I wondered if such command exists or if it is possible to add/create one. Sometimes I just want to automatize a portion of a run. Another example could be I want Automata to search for a specific support servant, enter battle but NOT play the stage.

**Comment by d7c0** [comment]
I don't think so, but you might be able to make it intentionally 'crash' to stop the script. I'm not really sure on how that's done, maybe just give it impossible commands where you want it to stop. LMK if you figure it out.
